manubot/cite/csl_item.py

```diff
@@ -275,3 +301,75 @@ def standardize_id(self):
 def assert_csl_item_type(x):
     if not isinstance(x, CSL_Item):
         raise TypeError(f"Expected CSL_Item object, got {type(x)}")
+
+
+def date_to_date_parts(iso_date):
```

Comment:
Would we ever need this as a util function outside of this class?

Reply:
Yes I think it's useful to have access to it without having to create a dummy CSL_Item, so I placed it outside the class.